contribute/style-guides/frontend.md

```diff
@@ -172,6 +172,44 @@ const CONSTANT_VALUE = "This string won't change";
 
 _SASS styles are deprecated. Please migrate to Emotion whenever you need to modify SASS styles._
 
+### Typing
+
+In general we let Typescript to infer the types so there is no need to type each variable. There are some exceptions to this:
```

Comment:
```suggestion
In general, you should let Typescript infer the types so that there's no need to explicitly define type for each variable. 

There are some exceptions to this:
```